src/components/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,20 @@
+import { useLocation, useMatches } from '@remix-run/react';
+import { RouteMatch } from '~/router/types';
+import styles from './breadcrumbs.module.scss';
+import { ArrowRightIcon } from '../icons';
+
+export const Breadcrumbs = () => {
+    const matches = useMatches() as RouteMatch[];
+    const location = useLocation();
+
+    return (
+        <div className={styles.breadcrumbs}>
+            {matches
+                .flatMap((match) => match.handle?.breadcrumb?.(match, location) ?? [])
+                .map((breadcrumb, index, arr) => [
+                    breadcrumb,
+                    index !== arr.length - 1 && <ArrowRightIcon className={styles.separatorIcon} />,
```

Comment:
No warning about missing `key` on children?

Reply:
No, which is weird. I've added React.Fragment with a key